app/Providers/LoadConfigServiceProvider.php

```diff
@@ -42,6 +42,25 @@ public function boot()
         $this->app->config->set('app.url', $appDomain ?: $this->app->config->get('app.url'));
         $this->app->config->set('app.locale', $appLocale ?: $this->app->config->get('app.locale'));
 
+        // Setup Cors.
+        $allowedOrigins = [
+            Setting::get('app_domain'),
```

Comment:
Remember to call Settings on the try catch if not we will have a failing test suite.